extensions/binding/org.eclipse.smarthome.binding.yahooweather/ESH-INF/thing/thing-types.xml

```diff
@@ -16,13 +16,13 @@
 		</channels>
 
         <config-description>
-            <parameter name="location" type="text" required="true">
+            <parameter name="location" type="integer" required="true">
                 <label>Location</label>
                 <description>Location for the weather information.
                     Syntax is WOEID, see https://en.wikipedia.org/wiki/WOEID.
                 </description>
             </parameter>
-            <parameter name="refresh" type="integer">
+            <parameter name="refresh" type="integer" min="1">
```

Comment:
The minimum of the refresh parameter is not related to the config status (or I missed something).